dev-client/src/translations/en.json

```
@@ -267,7 +267,7 @@
       "delete": "Delete this project",
       "delete_button": "Delete",
       "delete_button_prompt": "Delete Project?",
```

Nitpick looking at the Figma
```suggestion
      "delete_button_prompt": "Delete project?",
```